easybuild/easyconfigs/k/KAT/KAT-2.4.2-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'ConfigureMake'
+
+name = 'KAT'
+version = '2.4.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.earlham.ac.uk/kat-tools'
+description = """The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/TGAC/KAT/archive']
+sources = ['Release-%(version)s.tar.gz']
+patches = [
+    'KAT-%(version)s-use-EB-Boost.patch',
+    'KAT-%(version)s-fix-python.patch',
+]
+checksums = [
+    'd6116cefdb5ecd9ec40898dd92362afe1a76fa560abfe0f2cd29cbe0d95cb877',  # Release-2.4.2.tar.gz
+    'e8be718dc1c9668d6b551a019d9364e97199797dc485552e132295603342bb35',  # KAT-2.4.2-use-EB-Boost.patch
+    '566f38ced78a0d74a3740ec5913f624a2e6956011041947856164b79cfba44b3',  # KAT-2.4.2-fix-python.patch
+]
+
+builddependencies = [('Autotools', '20170619')]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Python', '3.6.4'),
+    ('matplotlib', '2.1.2', versionsuffix),
+    ('Boost', '1.66.0', versionsuffix),
+]
+
+preconfigopts = "./autogen.sh && "
+
+buildopts = "V=1"
```

Comment:
@wpoely86 Is this still needed?

Reply:
it's taste, else you don't see what is going on...

Reply:
We set `-DCMAKE_VERBOSE_MAKEFILE=ON` for `CMakeMake` (which implies `make V=1`), we should to the same for `ConfigureMake` rather than only do it for some easyconfigs?